pyodide-build/pyodide_build/cli/build.py

```diff
@@ -14,8 +14,6 @@
 from ..out_of_tree import build
 from ..out_of_tree.utils import initialize_pyodide_root
 
-app = typer.Typer()
```

Comment:
What is the significance of including/removing this? Doesn't this introduce the `build` subcommand?

Reply:
It has no effect. The `pyodide build` entrypoint directly uses [`main`](https://github.com/pyodide/pyodide/blob/4114f20986a3caa923d9bd512a3795a8b9044a49/pyodide-build/setup.cfg#L45) function. I think @joemarshall added this `app` variable in order to use it in tests. I changed it in a way that [typer docs](https://typer.tiangolo.com/tutorial/testing/#test-a-function) recommend, so it is not needed now.